FeatureDetect/root/source/featurelib.h

```diff
@@ -0,0 +1,18 @@
+#ifndef FEATUREDETECT_FEATURELIB_H_
+#define FEATUREDETECT_FEATURELIB_H_
+
+#include <vector>
+#include <iostream>
+#include <opencv2/opencv.hpp>
+#include <opencv2/core/core.hpp>
+#include <opencv2/features2d.hpp>
+#include <opencv2/imgproc.hpp>
+
+class featurelib  {
+public:
+     featurelib();
+private:
+     cv::Ptr<cv::Feature2D> sift = cv::SIFT::create();
+};
+
+#endif //FEATUREDETECT_FEATURELIB_H_
```

Comment:
Add a space between `//` and `FEATUREDETECT_FEATURELIB_H_` -- as I said before, it makes the comments more readable